src/install/kafka/install-infra.mdx

```diff
@@ -0,0 +1,6 @@
+---
+componentType: default
+headingText: Download the infrastructure agent
+---
+
+To use the Kafka integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Kafka-specific data such as database and instance metrics.
```

Comment:
Does the infra agent need to be installed first? If so we should probably replace "also" with "first"